### ISSUE TITLE
Longformer using BERT

### ISSUE THREAD

**Issue by antonioFlavio** [allenai/longformer#61]
Hi folks! First of all, Thank you a lot for sharing this code with the community!! And thank your for the hard work with contribution on NLP area.

Based on [convert_model_to_long.ipynb](https://github.com/allenai/longformer/blob/master/scripts/convert_model_to_long.ipynb), I'm trying to use BERT pt-br (from [neuralmind](https://github.com/neuralmind-ai/portuguese-bert)) version to generate a portuguese longformer model. After some ajusts at the original code, I'm stucked on an error, catched on method 'pretrain_and_evaluate' calling 'trainer.evaluate()':

forward() got an unexpected keyword argument 'labels'

Did you tried use any bert model as base? Do you know how I can move on about this error?

Thanks!

**Comment by antonioFlavio** [comment]
> I think it has to do with the use of DataCollatorForLanguageModelling.

Yes, it is.  The input array extracted from dataloader, has 2 coluns: input_ids and labels. Both has the same value. The bert model that I'm using, don't have the column 'labels' as a parameter.

I'm using BERT, so, to resolve that temporarily, I'm copy all [Trainer.py](https://github.com/huggingface/transformers/blob/master/src/transformers/trainer.py) code from transformer repo, and remove the 'labels' column before input into to the model from evaluate and train methods.

**Comment by ibeltagy** [comment]
Closing. Please feel free to reopen or create a new one if you have more questions.
